src/test/regress/expected/citus_schema_distribute_undistribute.out

```diff
@@ -0,0 +1,381 @@
+SELECT nspname AS temp_toast_schema_name FROM pg_namespace WHERE nspname LIKE 'pg_toast_temp%' LIMIT 1 \gset
+SELECT citus_schema_distribute(:'temp_schema_name');
+ERROR:  temporary schema cannot be distributed
+SELECT citus_schema_distribute(:'temp_toast_schema_name');
+ERROR:  temporary schema cannot be distributed
+SELECT citus_schema_distribute('citus');
+ERROR:  schema citus, which is owned by an extension, cannot be distributed
+-- weird schema and table names
+CREATE SCHEMA "CiTuS.TeeN";
+CREATE TABLE "CiTuS.TeeN"."TeeNTabLE.1!?!"(id int PRIMARY KEY, name text);
+INSERT INTO "CiTuS.TeeN"."TeeNTabLE.1!?!" SELECT i, 'asd'::text FROM generate_series(1,20) i;
+SELECT citus_schema_distribute('"CiTuS.TeeN"');
+ citus_schema_distribute
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT citus_schema_undistribute('"CiTuS.TeeN"');
+ citus_schema_undistribute
+---------------------------------------------------------------------
+
+(1 row)
+
+-- try setting the schema again after adding a distributed table into the schema. It should complain about distributed table.
+CREATE TABLE tenant1.new_dist(id int);
+SELECT create_distributed_table('tenant1.new_dist', 'id');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT citus_schema_distribute('tenant1');
+ERROR:  schema already has distributed tables
+HINT:  Undistribute distributed tables under the schema before distributing the schema.
+SELECT undistribute_table('tenant1.new_dist');
+ undistribute_table
+---------------------------------------------------------------------
+
+(1 row)
+
+-- try setting the schema again after adding a single shard table into the schema. It should complain about distributed table.
+CREATE TABLE tenant1.single_shard_t(id int);
+SELECT create_distributed_table('tenant1.single_shard_t', NULL);
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT citus_schema_distribute('tenant1');
+ERROR:  schema already has distributed tables
+HINT:  Undistribute distributed tables under the schema before distributing the schema.
+SELECT undistribute_table('tenant1.single_shard_t');
+ undistribute_table
+---------------------------------------------------------------------
+
+(1 row)
+
+-- try setting the schema again. It should succeed now.
+SELECT citus_schema_distribute('tenant1');
+ citus_schema_distribute
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT schemaid::regnamespace as schemaname FROM pg_dist_tenant_schema WHERE schemaid::regnamespace::text = 'tenant1';
+ schemaname
+---------------------------------------------------------------------
+ tenant1
+(1 row)
+
+SELECT colocationid = (SELECT colocationid FROM pg_dist_tenant_schema WHERE schemaid::regnamespace::text = 'tenant1')
```

Comment:
Nit, could use `all(colocationid = (SELECT colocationid FROM pg_dist_tenant_schema WHERE schemaid::regnamespace::text = 'tenant1'))` or such.

Can we also query pg_dist_colocation to check that the colocation id is really registered into pg_dist_colocation?